Fastfile

```diff
@@ -14,6 +14,15 @@ default_platform :ios
     scan(skip_slack: skip_slack)
   end
 
+  lane :ui_test do
+    _setup()
+    skip_slack = ENV['SCAN_SLACK_CHANNEL'].to_s.strip.empty?
+    scan(skip_slack: skip_slack,
+         devices: ["iPhone 8"],
+         output_types: "html",
+         scheme: ENV['UI_TEST_SCHEME'])
```

Comment:
following naming convention for env variables we have defined our selves can we update this to `UI_TEST_SCHEME `